src/components/HomeViewUsers/index.css

```diff
@@ -0,0 +1,38 @@
+.img__user {
+  height: 100px;
+  width: 100px;
+  border-radius: 50%;
+  border: 2px solid #04a9e0;
+}
+
+h2 {
+  font-family: "Arial", "Open Sans";
+  color: #828282;
+  text-transform: capitalize;
+}
+
+.view__users {
```

Comment:
this is not correct naming according to BEM.
We should start wit the component name.